### PR TITLE
Debug branch

### PR DESCRIPTION
**Pull request by RedPandore**

### CODE REVIEW

**Review by bouboumael — DISMISSED** [review]

File: templates/includes/_presentation.html.twig [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        <div class="row">
+            <div class="d-flex justify-content-center">
+                <ol>
+                    <li class="d-flex justify-content-start align-items-center list">
+                        <div class="mx-3 number">
+                            <p>1</p>
+                        </div>
+                        <div class="list-text">
+                            <div class="h1 list-title">Je me rends sur jobpermut.fr</div>
+                            <div class="h3 text-muted list-subtitle">C'est une première étape non ?</div>
```
Comment (LucyMerlier) [inline templates/includes/_presentation.html.twig]: erm... don't get me wrong, those little comments are obviously funny, but you're gonna potentially present this  to your client, so you should keep it professionnal

File: templates/includes/_presentation.html.twig [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+                        </div>
+                    </li>
+                    <li class="d-flex justify-content-start align-items-center list">
+                        <div class="mx-3 number">
+                            <p>2</p>
+                        </div>
+                        <div class="list-text">
+                            <div class="h1 list-title">Je m'inscris et rentre mes infos personnelles</div>
+                            <div class="h3 text-muted list-subtitle">Et çà, une seconde étape non ?</div>
```
Comment (LucyMerlier) [inline templates/includes/_presentation.html.twig]: same as above

File: templates/includes/_presentation.html.twig [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+                        </div>
+                    </li>
+                    <li class="d-flex justify-content-start align-items-center list">
+                        <div class="mx-3 number">
+                            <p>3</p>
+                        </div>
+                        <div class="list-text">
+                            <div class="h1 list-title">Je selectionne mon trajet quotidien</div>
+                            <div class="h3 text-muted list-subtitle">Si je ne m'abuse, c'est la 3ème celle là ?</div>
```
Comment (LucyMerlier) [inline templates/includes/_presentation.html.twig]: same as above

File: templates/includes/_presentation.html.twig [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+                        </div>
+                    </li>
+
+                    <li class="d-flex justify-content-start align-items-center list">
+                        <div class="mx-3 number">
+                            <p>4</p>
+                        </div>
+                        <div class="list-text">
+                            <div class="h1 list-title">Je match avec un autre utilisateur</div>
+                            <div class="h3 text-muted list-subtitle">La quaaaatrièèèèème ?</div>
```
Comment (LucyMerlier) [inline templates/includes/_presentation.html.twig]: same as above

File: templates/includes/_presentation.html.twig [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+                    </li>
+                    <li class="d-flex justify-content-start align-items-center list">
+                        <div class="mx-3 number">
+                            <p>
+                                <i class="fas fa-check"></i>
+                            </p>
+                        </div>
+                        <div class="list-text">
+                            <div class="h1 list-title">C'est bon !</div>
+                            <div class="h3 text-muted list-subtitle">C'est fini, plus d'étape ...</div>
```
Comment (LucyMerlier) [inline templates/includes/_presentation.html.twig]: same as above

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comments

**Review by bouboumael — DISMISSED** [review]

File: templates/includes/_presentation.html.twig (hunk 1 above)
Comment (RedPandore) [reply templates/includes/_presentation.html.twig]: Not funny ....

File: templates/includes/_presentation.html.twig [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+                            <div class="h3 mt-0 text-muted list-subtitle">Je peux valider le match et voir ceux qu'on me propose</div>
+                        </div>
+                    </li>
+                    <li class="d-flex justify-content-start align-items-center list">
+                        <div class="mx-3 number">
+                            <p>5</p>
+                        </div>
+                        <div class="list-text">
+                            <div class="h1 mb-0 list-title">Le match est accepté par les deux utilisateurs</div>
+                            <div class="h3 mt-0 text-muted list-subtitle">Notre dossier est analysée par l'équipe JobPermut</div>
```
Comment (LucyMerlier) [inline templates/includes/_presentation.html.twig]: typo : "analysé" * (because "dossier" is masculine)

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comment

**Review by bouboumael — APPROVED** [review]

**Review by LucyMerlier — APPROVED** [review]

**Pull request merged**